src/sentry/static/sentry/app/types/hooks.tsx

```diff
@@ -53,8 +53,8 @@ type SelectorItemsProps = React.ComponentProps<typeof SelectorItems>;
  * Component wrapping hooks
  */
 export type ComponentHooks = {
-  'component:header-date-range': React.ComponentType<DateRangeProps>;
-  'component:header-selector-items': React.ComponentType<SelectorItemsProps>;
+  'component:header-date-range': () => React.ComponentType<DateRangeProps>;
+  'component:header-selector-items': () => React.ComponentType<SelectorItemsProps>;
```

Comment:
Is this weirdly inconsistent?

Reply:
It sure is! But that is the interface that `HookOrDefault` wants. 

https://github.com/getsentry/sentry/blob/d830eb47b4c60ae45994464e157587b5880cce53/src/sentry/static/sentry/app/components/hookOrDefault.tsx#L96-L102

`HookOrDefault` will call this function with hook parameters, but they are unused here.

Reply:
Do you know why this wasn’t producing a type error in sentry before?

Reply:
I'm guessing that the `any` coming from getsentry made it 'ok'

Reply:
Hm, this feels a little bit concerning since the hook types in sentry should be independent of getsentry